packages/terra-consumer-nav/src/components/quick-links/QuickLinks.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import Button from 'terra-button';
+import styles from './QuickLinks.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  quickLinks: PropTypes.arrayOf(
```

Comment:
make a `QuickLink` component that takes the properties and create an array of those insetad.